src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -494,7 +494,7 @@ public void TestEnvironmentVariables_Environment_DataRoundTrips()
             Assert.False(psi.Environment.Contains(new KeyValuePair<string, string>("NewKey3", "NewValue3")));
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))] // ActiveIssue: https://github.com/dotnet/corefx/issues/29753
```

Comment:
We can treat this as by design for Nano so please don't bother with the ActiveIssue. Something like `// Nano does not support these verbs` is fine